### ISSUE TITLE
Provide a way to display reactions

### ISSUE THREAD

**Issue by brettcannon** [microsoft/vscode-github-issue-notebooks#55]
It would be great if there was a way to show reactions for an issue. Specifically, we would like to know what the top voted issues along with their vote count.

**Comment by jrieken** [comment]
Not sure if the data the queries returns contains this information. Otherwise we would need to fetch this per issue...

**Comment by brettcannon** [comment]
Nope, it's not included in the default REST API payload for an issue. But it looks like it's in preview with the `application/vnd.github.squirrel-girl-preview` API: https://docs.github.com/en/rest/reference/issues#get-an-issue.
